### DISCUSSION TITLE
pulsar-broker pod个数只能1个吗？

### DISCUSSION THREAD

**Discussion by un-human** [milvus-io/milvus#38294] [Q&A and General discussion]
milvus==2.2.3，helm安装cluster；pulsar-broker 的pod个数只能是1个吗？是否可以2个pod个数？

**Comment by yhmo** [comment]
https://milvus.io/docs/v2.2.x/install_cluster-milvusoperator.md
这文档里的截图里，pulsar-broker不就是2个么

**Reply by un-human** [reply]
> https://milvus.io/docs/v2.2.x/install_cluster-milvusoperator.md 这文档里的截图里，pulsar-broker不就是2个么

嗯，特意看了一下pulsar,可以直接扩
![image](https://github.com/user-attachments/assets/708e3788-8722-494e-af07-d423b610c8c2)
